site/en/faq.md

```diff
@@ -91,7 +91,7 @@ tool offers simple ways to create new METS files or augment existing ones.
 OCR-D is Free Software, licensed under the terms of the Apache 2.0 license and
 will be free to use and adapt in perpetuity.
 
-### How are the full texts produced by OCR-D presented to the (library) user? Are they integrated into the library catalogue and can therefore be used for full text search in the catalogue?
+### How are the full texts produced by OCR-D presented to the (library) user? Are they integrated into the library catalog and can therefore be used for full text search in the catalog?
```

Comment:
The US form `catalog` is more common than the English `catalogue`. I kept the latter in names like the Union Catalogues of Books.